dev/Tiles/metatile.h

```diff
@@ -0,0 +1,60 @@
+#include <stdbool.h>
+#include <stdlib.h>
+#include "../../lib/SMSlib.h"
+#include "tile.h"
+
+typedef struct MetaTile
+{
+    struct Tile tiles[16];
```

Comment:
Could have just used your typedef here?

```suggestion
    Tile tiles[16];
```

Reply:
Yeah, some tutorials described it like that and it worked so I didn't poke the bear. I did notice that it also works without. I'll remove them.